src/components/TextArea.tsx

```diff
@@ -125,10 +125,29 @@ export class TextArea extends React.Component<TextAreaProps, TextAreaState> {
       });
   };
 
+  loadEmptySuggestion = (target: HTMLTextAreaElement, key: string) => {
+    const caret = getCaretCoordinates(target, key);
+    this.startLoadingSuggestions("");
+    this.setState({
+      mention: {
+        status: "loading",
+        startPosition: target.selectionStart + 1,
+        caret: caret,
+        suggestions: [],
+        triggeredBy: key
+      }
+    });
+  }
```

Comment:
Should there not be tests for this function or these changes?

How come we cannot reuse the javascript-mention logic we already have in the user-facing application? It may need updating, but if it was re-used we'd assure consistent behavior, right?